src/__tests__/commands/apps.test.ts

```diff
@@ -4,25 +4,18 @@ import { ArgumentsCamelCase, Argv, Options } from 'yargs'
 
 import { AppClassification, AppResponse, AppType, AppsEndpoint, PagedApp, SmartThingsClient } from '@smartthings/core-sdk'
 
-import { APICommand, APICommandFlags, apiCommand, apiCommandBuilder, apiDocsURL } from '../../lib/command/api-command.js'
+import { APICommand, APICommandFlags } from '../../lib/command/api-command.js'
 import { outputItemOrList, outputItemOrListBuilder } from '../../lib/command/listing-io.js'
 import { CommandArgs } from '../../commands/apps.js'
 import { ListDataFunction } from '../../lib/command/io-defs.js'
 import { BuildOutputFormatterFlags } from '../../lib/command/output-builder.js'
 import { SmartThingsCommandFlags } from '../../lib/command/smartthings-command.js'
 import { shortARNorURL, verboseApps } from '../../lib/command/util/apps-util.js'
+import { apiCommandMocks } from '../test-lib/api-command-mock.js'
 import { buildArgvMock, buildArgvMockStub } from '../test-lib/builder-mock.js'
 
 
-
-const apiCommandMock = jest.fn<typeof apiCommand>()
-const apiCommandBuilderMock = jest.fn<typeof apiCommandBuilder>()
-const apiDocsURLMock = jest.fn<typeof apiDocsURL>()
-jest.unstable_mockModule('../../lib/command/api-command.js', () => ({
-	apiCommand: apiCommandMock,
-	apiCommandBuilder: apiCommandBuilderMock,
-	apiDocsURL: apiDocsURLMock,
-}))
+const { apiCommandMock, apiCommandBuilderMock } = apiCommandMocks('../..')
```

Comment:
I don't love that we need to pass in the directory prefix here but `jest.unstable_mockModules` works relative to the test itself. There might be a way to determine this in the module but I haven't found anything yet.

Reply:
Thinking about this for a nanosecond longer and I'm realizing we probably do have more commands at one level deep overall so maybe we should make a default of `../../..`? 🤷